modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -92,3 +92,55 @@ spec:
 ----
 <1> Enables the monitoring tab in the Jaeger console.
 <2> The service name for Thanos Querier from user-workload monitoring.
+
+== Span RED metrics and alerting rules
+
+The metrics generated by the `spanmetrics` connector are usable with alerting rules. For example, for alerts about a slow service or to define service level objectives (SLOs), the connector creates a `duration_bucket` histogram and the `calls` counter metric. These metrics have labels that identify the service, API name, operation type, and other attributes.
+
+.Labels of the metrics created in the `spanmetrics` connector
+[options="header"]
+[cols="l, a, a"]
+|===
+|Label |Description |Values
+
+|service_name
+|Service name set by the `otel_service_name` environment variable.
+|`frontend`
+
+|span_name
+| Name of the operation.
+|
+* `/`
+* `/customer`
+
+|span_kind
+|Identifies the server, client, messaging, or internal operation.
```

Comment:
Can drop the punctuation on the 3 "Description" sentences

Reply:
Thank you, I'll look into this.